build-pipeline/windows-pipeline.groovy

```diff
@@ -18,6 +18,9 @@ def buildImages = {
 if (params.os == '1709') {
     simpleNode('windows.10.amd64.serverrs3.open', buildImages)
 }
+else if (params.os == '1803') {
+    simpleNode('windows.10.amd64.serverrs4.open', buildImages)
```

Comment:
Does this exist?  I thought it was still being created - https://github.com/dotnet/core-eng/issues/3365

Reply:
Is this the right way to identify the RS4 nodes? 

I thought I saw comments that this was ready to go: https://github.com/dotnet/core-eng/issues/3365#issuecomment-389995754.